Tests/Shared/VIMVideoTests.swift

```diff
@@ -198,4 +198,19 @@ class VIMVideoTests: XCTestCase {
         let testVideoObject = VIMVideo(keyValueDictionary: videoDictionary)!
         XCTAssertFalse(testVideoObject.isPrivate(), "Test video object is stock and should not return as private.")
     }
+
+    func test_videoAppProperty_isParsedAsExpected() {
```

Comment:
If you change this to `throws`:
```suggestion
    func test_videoAppProperty_isParsedAsExpected() throw {
```
then the `try!` can be changed to `try`.